src/views/About.jsx

```diff
@@ -8,53 +8,68 @@ export function About() {
 				<section className="About-section">
 					<img width="250px" alt="phone screenshot" src="../img/phone.png" />
 					<div className="About-section-text">
-						<h2>Make some lists</h2>
+						<h2>Lending a list-ening ear</h2>
 						<p>
-							Level up your organization game by creating specialized lists for
-							all your needs.
+							Aisle Be There to level up your organization game! Create
+							specialized lists for all your kneads!{' '}
+							<span role="img" aria-label="bread">
+								🍞{' '}
+							</span>
```

Comment:
🔍 Love this!  You could make a tiny little helper component to deduplicate all the `role="img"` and `aria-label`-ing:

```jsx
/**
 * Render an emoji with an accessible label.
 * @param {{ children: React.ReactNode, label: string }} props 
 */
function Emoji({ children, label }) {
	return (
		<span role="img" aria-label={label}>
			{children}
		</span>
	);
}

<Emoji label="Bread">🍞</Emoji>
```